jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -130,6 +141,28 @@
    */
   public static final String JAEGER_DISABLE_GLOBAL_TRACER = JAEGER_PREFIX + "DISABLE_GLOBAL_TRACER";
 
+  /**
+   * Comma separated list of formats to use for propagating the trace context. Default will the
+   * standard Jaeger format. Valid values are jaeger and b3.
+   */
+  public static final String JAEGER_PROPAGATION = JAEGER_PREFIX + "PROPAGATION";
+
+  /**
+   * The supported trace context propagation formats.
+   */
+  public enum Propagation {
+
+    /**
+     * The default Jaeger trace context propagation format.
+     */
+    jaeger,
```

Comment:
does this match Java convention for constants? I thought they should be upper case.